e2e/test/DeviceTokenRefreshE2ETests.cs

```diff
@@ -116,7 +116,14 @@ private async Task DeviceClient_TokenIsRefreshed_Internal(Client.TransportType t
 
                 var message = new Client.Message(Encoding.UTF8.GetBytes("Hello"));
 
-                using (var cts = new CancellationTokenSource(TimeSpan.FromSeconds(ttl * 10)))
+                // calculating test run timeout: 
+                // we use the ttl + max percentage buffer time * number of calls we make to WaitForTokenRefreshAsync
+                // and add a buffer for opening and sending on the deviceclient.
+                const int operationCompletionBufferTimeInSeconds = 10;
+                const int numberOfWaitForTokenCalls = 2;
+                var testRunTimeout = (numberOfWaitForTokenCalls * (ttl * (float)(1 + buffer / 100))) + operationCompletionBufferTimeInSeconds;
```

Comment:
Nit: `int` instead of `var` (see coding guidelines on using var only when the type is written on the same line.

Reply:
Ok. Fixed.